### ISSUE TITLE
Error opening memory series storage: file missing" source="main.go:182 : Error message is not explicit

### ISSUE THREAD

**Issue by akram** [prometheus/prometheus#2888]
[NOTICE]: <> (If your question is around usage and not a bug in Prometheus please use: https://groups.google.com/forum/#!forum/prometheus-users)

**What did you do?**
We maybe have data corruption, we endup in this message which is not very helpful.

**What did you expect to see?**
Since the ErrMissingFiles struct has    Fds []storage.FileDesc we want to have a list of the missing files so we can troubleshoot or restore missing files if relevant.

**What did you see instead? Under which circumstances?**
The simple error message: " file missing"

* Prometheus version: 

version=1.5.2, branch=master, revision=bd1182d29f462c39544f94cc822830e1c64cf55b

* Logs:
```
revision=bd1182d29f462c39544f94cc822830e1c64cf55b)" source="main.go:75"
time="2017-06-29T09:50:51Z" level=info msg="Build context (go=go1.7.5, user=root@1a01c5f68840, date=20170210-16:23:28)" source="main.go:76"
time="2017-06-29T09:50:51Z" level=info msg="Loading configuration file /etc/prometheus/prometheus.yml" source="main.go:248"
time="2017-06-29T09:50:52Z" level=error msg="Error opening memory series storage: file missing" source="main.go:182"
```

**Comment by akram** [comment]
fixed by #2889

**Comment by beorn7** [comment]
This is a LevelDB problem and needs to be fixed upstream.

The on-disk format of LevelDB is standardized, however, so you could shut down the Prometheus server and apply 3rd party LevelDB recovery and diagnostics tools on the LevelDB files.
